akka-actor-tests/src/test/scala/akka/io/dns/AsyncDnsResolverSpec.scala

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.io.dns
+
+import java.net.InetAddress
+
+import akka.io.dns.DnsProtocol.{ Ip, RequestType, Srv }
+import akka.io.{ Dns, IO }
+import akka.pattern.ask
+import akka.testkit.AkkaSpec
+import akka.util.Timeout
+
+import scala.concurrent.duration._
+
+//TODO: Setup a DNS server on our jenkins hosts
```

Comment:
need to address this or not really? keeping the todo or not?

Reply:
I am going to create a stubbed test for now before I merge this